### PR TITLE
rr: update to 5.2.0.

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: srcpkgs/rr/template [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 short_desc="Nondeterministic debugger"
 maintainer="Spencer Hill <spencernh77@gmail.com>"
-license="MIT, BSD"
+license="MIT, BSD-2-Clause"
 homepage="http://rr-project.org/"
-distfiles="https://github.com/mozilla/${pkgname}/archive/${version}.tar.gz"
-checksum=2388ee4a16dafcb21002e93cb9afa7f49c13e8932e4fd110c82a7f19d6ec956b
+distfiles="https://github.com/mozilla/rr/archive/${version}.tar.gz"
+checksum=a15900fd4c49f360e678542f00742b3cd7811bdf4bab2ca3fd38cae932183a37
+only_for_archs="i686 x86_64"
```
Comment (Gottox) [inline srcpkgs/rr/template]: musl doesn't work?

**Comment by maxice8** [comment]
On Mon, May 28, 2018 at 12:52:09AM -0700, Enno Boland wrote:
> Gottox commented on this pull request.
> 
> 
> 
> >  homepage="http://rr-project.org/"
> -distfiles="https://github.com/mozilla/${pkgname}/archive/${version}.tar.gz"
> -checksum=2388ee4a16dafcb21002e93cb9afa7f49c13e8932e4fd110c82a7f19d6ec956b
> +distfiles="https://github.com/mozilla/rr/archive/${version}.tar.gz"
> +checksum=a15900fd4c49f360e678542f00742b3cd7811bdf4bab2ca3fd38cae932183a37
> +only_for_archs="i686 x86_64"
> 
> musl doesn't work?
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/voidlinux/void-packages/pull/14741#pullrequestreview-123651087

no clue, i didn't check.

**Comment by Johnnynator** [comment]
It will not work without additional work

```
==> rr__do_configure.log
-- The C compiler identification is GNU 7.3.0
-- The CXX compiler identification is GNU 7.3.0
-- The ASM compiler identification is GNU
-- Found assembler: /usr/lib/ccache/bin/cc
-- Check for working C compiler: /usr/lib/ccache/bin/cc
-- Check for working C compiler: /usr/lib/ccache/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/lib/ccache/bin/g++
-- Check for working CXX compiler: /usr/lib/ccache/bin/g++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-musl/7.3.0/libgcc.a when searching for -lgcc
/usr/bin/ld: cannot find -lgcc
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-musl/7.3.0/../../../../lib/libgcc_s.so.1 when searching for libgcc_s.so.1
/usr/bin/ld: skipping incompatible /lib/../lib/libgcc_s.so.1 when searching for libgcc_s.so.1
/usr/bin/ld: skipping incompatible /usr/lib/../lib/libgcc_s.so.1 when searching for libgcc_s.so.1
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-musl/7.3.0/../../../libgcc_s.so.1 when searching for libgcc_s.so.1
/usr/bin/ld: skipping incompatible /lib/libgcc_s.so.1 when searching for libgcc_s.so.1
/usr/bin/ld: skipping incompatible /usr/lib/libgcc_s.so.1 when searching for libgcc_s.so.1
/usr/bin/ld: cannot find libgcc_s.so.1
/usr/bin/ld: skipping incompatible /usr/lib/gcc/x86_64-linux-musl/7.3.0/libgcc.a when searching for -lgcc
/usr/bin/ld: cannot find -lgcc
collect2: error: ld returned 1 exit status
CMake Error at CMakeLists.txt:97 (message):
  Your toolchain doesn't support 32-bit cross-compilation.  Install the
  required packages or pass -Ddisable32bit=ON to cmake.


-- Configuring incomplete, errors occurred!
See also "/builddir/rr-5.2.0/build/CMakeFiles/CMakeOutput.log".
See also "/builddir/rr-5.2.0/build/CMakeFiles/CMakeError.log".
[1m[31m=> ERROR: rr-5.2.0_1: do_configure: 'cmake ${cmake_args} ${configure_args} $(echo ${cmake_builddir}|sed 		-e 's|[^/]$|/|' -e 's|[^/]*||g' -e 's|/|../|g')' exited with 1
[m[1m[31m=> ERROR:   in do_configure() at common/build-style/cmake.sh:38
[m

```

**Comment by Johnnynator** [comment]
Just passing ` -Ddisable32bit=ON` to cmake is not enough. Consider it broken on musl for now.

**Pull request closed without merging**